esprima.js

```diff
@@ -205,7 +214,12 @@
         DuplicateProtoProperty: 'Duplicate __proto__ fields are not allowed in object literals',
         ConstructorSpecialMethod: 'Class constructor may not be an accessor',
         DuplicateConstructor: 'A class may only have one constructor',
-        StaticPrototype: 'Classes may not have static property named prototype'
+        StaticPrototype: 'Classes may not have static property named prototype',
+        MissingFromClause: 'Missing from clause',
```

Comment:
Ok, then seems like we want to convert these all into "unexpected token" messages? Or should we go with these for now, @ariya?